src/less/widgets/widgets-general.less

```diff
@@ -38,6 +38,12 @@
     color: @text;
   }
 
+
+  .pf-widget-footer {
```

Comment:
curious if this should be `display:none` by default as it mixed with the slideout style above might effect widgets in production since they'd be missing the necessary override of the `pf-widget-footer`. might be worth testing against some variations in production to make sure?